Lesson9/Gokhan/styles.css

```diff
@@ -0,0 +1,104 @@
+
+/* i added text content, pre element and content centeren on page */
+/* when i was write and save */
+/* i can^t use live server for this work */
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 100%;
+  display:flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 100px auto;
+  font-family: source-serif-pro, Georgia, Cambria, "Times New Roman", Times, serif;
+}
+
+.text-content {
```

Comment:
Removed this class from the body element. It could be appleid to the main element instead but important not to forget margin ' margin: 100px auto;' that will center content on the page.